core/hybrisadaptor.cpp

```diff
@@ -226,19 +227,10 @@ void HybrisManager::stopReader(HybrisAdaptor *adaptor)
 
 bool HybrisManager::resumeReader(HybrisAdaptor *adaptor)
 {
-    sensordLogD() << Q_FUNC_INFO << adaptor->id() << adaptor->deviceStandbyOverride(); //alwaysOn
-    QList <HybrisAdaptor *> list;
-    list = registeredAdaptors.values();
-    bool okToResume = false;
-    for (int i = 0; i < list.count(); i++) {
-        if (list.at(i) != adaptor && list.at(i)->isRunning()) {
-            qDebug() << Q_FUNC_INFO << "running";
-            okToResume = true;
-        }
-    }
-
-    if (okToResume) {
-        sensordLogD() << "activating for resume" << adaptor->name();
+    sensordLogD() << Q_FUNC_INFO << adaptor->id() << adaptor->deviceStandbyOverride();
+    if (!adaptor->deviceStandbyOverride()) {
+        setDelay(adaptor->sensorHandle, adaptor->interval() * 1000000);
+    } else {
```

Comment:
See my comment about standbyOverride. it is a property that tells sensorfw that the client wants to remain running while screen is blanked.